rclpy/test/test_executor.py

```diff
@@ -577,6 +579,41 @@ def test_multi_threaded_spin_once_until_future(self):
 
         executor.shutdown()
 
+    def test_not_lose_callback(self):
+        # This test is from issue https://github.com/ros2/rclpy/issues/1159
+
```

Comment:
I think we can remove this link since this PR links the original issue, so that we can trace the history.

```suggestion
```

Reply:
Okay.